client/master.go

```diff
@@ -202,8 +203,9 @@ func (m *masterClient) QueryRouter(ctx context.Context, key string) ([]string, e
 	}
 	routerIPs := make([]string, 0, len(bytesRouterIP))
 	for _, bs := range bytesRouterIP {
-		routerIPs = append(routerIPs, string(bs))
-		log.Debugf("find key: [%s], routerIP: [%s]", key, string(bs))
+		ip := strings.Split(string(bs), ":")[0]
```

Comment:
The change in this place is correct because the metadata has changed. But it is not related to the logic of auto recover ps, this place will not be actively called.